Controller/ContentManagement/CrudController.php

```diff
@@ -358,4 +374,50 @@ public function getContentTypeInfo(ContentType $contentType)
                 'contentType' => $contentType,
         ]);
     }
+
+
+    /**
+     * @return array{username:string, displayName:string, roles:array<string>, email:string, circles:array<string>}
+     */
+    private function userToApiUserArray(User $user): array
```

Comment:
Small detail, public functions first private after. This should be moved to the bottom

Reply:
I still think this function should be part of the User object "$user->toApiArray()" The CrudController should not need to care about how to construct this array

Reply:
moved c5d0e467c0d7786047dff7c40f41a1f4c26cca4c

Reply:
even better in the user entity fda1219a36c1bec8f610426fdb8efc358057a658